### DISCUSSION TITLE
LibMesh Update 2022.10.05

### DISCUSSION THREAD

**Discussion by cticenhour** [idaholab/moose#22321] [News]
A new libMesh has made its way to the MOOSE master branch. For those using the conda packages, please perform
```
mamba update --all
```
in your MOOSE conda environment. The appropriate `moose-libmesh` package version should be `2022.10.05`. For those using the libMesh submodule, please run in the moose/scripts directory
```
./update_and_rebuild_libmesh.sh
```
after updating your copy of MOOSE.

**Comment by EdSheltonUKAEA** [comment]
Thank you for posting this information here.
I tried to do my regular MOOSE update yesterday, which failed.  I knew from previous experience that this is often because of an update to libmesh.  So I tried my usual method to update that which is:
`mamba update moose-libmesh=2022.10.05`
But MOOSE now doesn't seem to run or update at all.  It seems to be broken.
When I enter "mamba update --all", I get the following:
`Traceback (most recent call last):
  File "/home/eshelton/mambaforge3/bin/mamba", line 11, in <module>
    sys.exit(main())
  File "/home/eshelton/mambaforge3/lib/python3.9/site-packages/mamba/mamba.py", line 914, in main
    from conda.common.compat import ensure_text_type, init_std_stream_encoding
ImportError: cannot import name 'init_std_stream_encoding' from 'conda.common.compat' (/home/eshelton/mambaforge3/lib/python3.9/site-packages/conda/common/compat.py)`

Can you advise what I need to do to recover MOOSE with the latest version of libmesh?

**Reply by GiudGiud** [reply]
It looks like mamba is having issues.
I d remove the moose environment, remove mamba, then reinstall from scratch
https://mooseframework.inl.gov/getting_started/installation/conda.html

**Reply by EdSheltonUKAEA** [reply]
All seems to be reinstalled and running again now.  Many thanks for your help.

**Reply by nwozniak70** [reply]
I know I'm a little late but I had this issue last month and for me the reason was that mamba was out of date with conda because running `conda update --all` within the MOOSE environment doesn't update mamba, so my mamba version was way outdated compared to everything else and it stopped working. The suggested fix was to just update mamba in the base environment, which is not mentioned in the MOOSE install/update instructions or troubleshooting page, but it worked perfectly and I didn't have to reinstall anything. Figured this might help someone else in the future.
